### PR TITLE
DESCW-2452 Updated alphav3-1.json, and created new header and footer …

### PR DESCRIPTION
**Pull request by ShawnTurple**
…templates for v3.1

### CODE REVIEW

File: styles/alphav3-1.json [hunk 1]
```diff
@@ -167,24 +180,38 @@
     "custom": {}
   },
   "styles": {
+    "spacing": {
+      "padding": {
+        "top": "0px",
+        "right": "inherit",
+        "bottom": "0px",
+        "left": "inherits"
```
Comment (mhaswell-bcgov) [inline styles/alphav3-1.json]: "inherits" or "inherit"?

Comment (ShawnTurple) [reply styles/alphav3-1.json]: haha, obviously it doesn't seem like it was doing anything.

File: styles/alphav3-1.json [hunk 2]
```diff
@@ -34,15 +35,25 @@
...
           "color": "#2D2D2D",
           "name": "Text Colour"
         },
         {
-          "slug": "tertiary",
-          "color": "#255A90",
-          "name": "Link – Tertiary"
+          "slug": "link",
+          "color": "#1A5A96",
+          "name": "Link Color"
```
Comment (mhaswell-bcgov) [inline styles/alphav3-1.json:51]: Minor, but other `name`s use the Canadian spelling of color.

**Review by mhaswell-bcgov — DISMISSED** [review]

File: styles/alphav3-1.json (hunk 2 above)
Comment (ShawnTurple) [reply styles/alphav3-1.json:51]: i changed them all to color

**Review by mhaswell-bcgov — APPROVED** [review]

**Pull request merged**